bowling/tests/bowling_tests_gt.cpp

```diff
@@ -0,0 +1,200 @@
+#include "gtest/gtest.h"
+
+#include "Game.hpp"
+
+#include <filesystem>
+#include <fstream>
+
+class GameTests : public ::testing::Test {
+   protected:
+    Game game;
+    std::ofstream file;
+    std::string filePath;
+
+    void SetUp() override {
+        std::string fileName = "/test_file.txt";
+        filePath = std::filesystem::current_path().string() + fileName;
+        file.open(filePath, std::ios_base::out);
+    }
+
+    void TearDown() override {
+        file.close();
+        std::filesystem::remove(filePath);
+    }
+};
+
+using GameDeathTests = GameTests;
+
+TEST_F(GameDeathTests, GameCrashesWhenLoadedFromNonExistingFile) {
+    std::string fileName{"hiashdksjd.txt"};
+    std::string filePath = std::filesystem::current_path().string() + fileName;
+    std::string expectedMessage = "file " + filePath + " could not be opened!";
+    EXPECT_DEATH(game.loadFromFile(filePath), expectedMessage);
+}
```

Comment:
it's not a good behavior. Game should not crash. It can throw an exception or raise an error in a different way - return an error code. Noone expect crashing software. Crash is not a normal case.